pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
+
```

Comment:
What's the reason for this? The file is not executable.

Reply:
I added this to make shellcheck happy.